modules/recruiting/storage/main.tf

```
@@ -0,0 +1,62 @@
+resource "azurerm_resource_group" "tikjob_rg" {
+  name     = "tikjob-${var.env_name}-rg"
+  location = var.resource_group_location
+}
+
+resource "random_password" "db_password" {
+  length           = 30
+  special          = true
+  override_special = "_%@"
+}
+
+resource "azurerm_mysql_server" "tikjob_mysql" {
+  name                = "tikjob-ghost-database"
```

I think we need the env name here. Also making it consistent with the tikweb module.
```suggestion
  name                = "tikjob-${var.env_name}-mysql-server"
```

---

Done in https://github.com/Tietokilta/tietokilta.fi-infra/pull/12/commits/51fe870d36f230e6b80a6c19f46f3539ee195c39